cmd/samples/recipes/searchattributes/searchattributes_workflow.go

```diff
@@ -0,0 +1,175 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"go.uber.org/cadence/activity"
+	"strconv"
+	"time"
+
+	"github.com/uber-common/cadence-samples/cmd/samples/common"
+	"go.uber.org/cadence/.gen/go/shared"
+	"go.uber.org/cadence/client"
+	"go.uber.org/cadence/workflow"
+	"go.uber.org/zap"
+)
+
+/**
+ * This sample shows how to use search attributes. (Note this feature only work with ElasticSearch)
```

Comment:
This feature only works when `Cadence server is configured with ElasticSearch` .. I think wherever we add such a comment on the client and usage we should make clear that this points to server ES..